packages/core/src/lib/Select/SelectBeta/SelectBeta.styles.ts

```diff
@@ -167,7 +167,7 @@ const DivStyledOverlay = styled.div`
     bottom: 0;
     display: none;
     left: 0;
-    position: absolute;
+    position: fixed;
```

Comment:
is this correct? wont the overlay be fixed to the page instead of the parent? does this casue an issue when you scroll the page?

Reply:
the overlay is used to close the select menu if someone clicks outside of the select.

If absolute is used it takes the parent size which is usually not the screen size so the menu won't be closed. (checkout select component in admin playground it won't close even if clicked outside)

But if fixed is used it take up entire screen and if user clicks anywhere the select component will be closed